### PR TITLE
[CLOUD-2514] Make description of JAVA_MAX_MEM_RATIO environment variable container runtime-agnostic. Same for README.adoc

### PR DESCRIPTION
**Pull request by iankko**
Also, while at this, squeeze the double space character in descriptions of certain environment variables into single one

<b>Note:</b> Since this PR [depends on ```cct_module``` change for CLOUD-2413 made by Jonathan](https://github.com/jboss-openshift/cct_module/pull/222), it can be merged only <b>after</b> the [CLOUD-2413](https://issues.jboss.org/browse/CLOUD-2413) changes have been merged.
    
Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull request does not include fixes for other issues than the main ticket
- [ ] Attached commits represent unit of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@redhat.com>` - use `git commit -s`

### CODE REVIEW

**Comment by iankko** [comment]
@rcernich @Crumby PTAL (once got a chance)

Thank you, Jan

File: datagrid-openshift/module.yaml [hunk 1]
```diff
@@ -5,3 +5,10 @@ description: Legacy datagrid-openshift script package.
 execute:
 - script: configure.sh
   user: '185'
+envs:
+    - name: GC_MAX_METASPACE_SIZE
```
Comment (rcernich) [inline datagrid-openshift/module.yaml]: These should get pulled in through the Java module (os-java-run)

File: dynamic-resources/module.yaml [hunk 2]
```diff
@@ -4,3 +4,16 @@ version: '1.0'
...
 execute:
 - script: install.sh
+envs:
+    - name: "INITIAL_HEAP_PERCENT"
+      example: 0.5
+      description: Deprecated. See JAVA_INITIAL_MEM_RATIO.
+    - name: "CONTAINER_HEAP_PERCENT"
+      example: 0.5
+      description: Deprecated. See JAVA_MAX_MEM_RATIO.
+    - name: JAVA_MAX_MEM_RATIO
```
Comment (rcernich) [inline dynamic-resources/module.yaml]: These should get pulled in from the Java module, os-java-run.

File: os-amq-launch/module.yaml [hunk 3]
```diff
@@ -8,3 +8,8 @@ execute:
 artifacts:
 - path: ce-amq-drain-1.0.0.Final-redhat-1.jar
   md5: 65b9aba8e5a0b7d7b259a6028f85595e
+
+envs:
+    - name: "JAVA_OPTS_APPEND"
```
Comment (rcernich) [inline os-amq-launch/module.yaml]: I think this should be moved to the Java module, even though it's not used there directly.  Eventually it should be though, and it directly applies to Java options/JVM configuration.

File: os-datavirt-s2i/module.yaml [hunk 4]
```diff
@@ -4,3 +4,7 @@ version: '1.0'
 description: Legacy os-datavirt-s2i script package.
 execute:
 - script: prepare.sh
+envs:
+- name: CUSTOM_INSTALL_DIRECTORIES
```
Comment (rcernich) [inline os-datavirt-s2i/module.yaml]: Should this be part of os-eap-s2i?

File: os-eap64-openshift/module.yaml [hunk 5]
```diff
@@ -6,3 +6,10 @@ execute:
 - script: prepare.sh
 - script: configure.sh
   user: '185'
+envs:
+    - name: GC_MAX_METASPACE_SIZE
```
Comment (rcernich) [inline os-eap64-openshift/module.yaml]: These should get pulled in from the Java module, os-java-run.

File: os-eap7-openshift/module.yaml [hunk 6]
```diff
@@ -6,3 +6,10 @@ execute:
 - script: prepare.sh
 - script: configure.sh
   user: '185'
+envs:
+    - name: GC_MAX_METASPACE_SIZE
```
Comment (rcernich) [inline os-eap7-openshift/module.yaml]: os-java-run.

File: os-sso71/module.yaml [hunk 7]
```diff
@@ -5,7 +5,18 @@ description: Legacy os-sso71 script package.
...
 execute:
 - script: configure.sh
+
+run:
+      user: 185
+      cmd:
+          - "/opt/eap/bin/openshift-launch.sh"
+
+envs:
+    - name: "CLI_GRACEFUL_SHUTDOWN"
```
Comment (rcernich) [inline os-sso71/module.yaml]: Do we want to define this in os-eap-launch and remove it from the other modules?

Comment (iankko) [reply os-sso71/module.yaml]: Jonathan's change above introduced the ```os-eap-launch``` module. But per code review (```runServer``` routine IIRC), the ```CLI_GRACEFUL_SHUTDOWN``` variable is supported only by EAP 7.x (and derived) images, thus used ```os-eap7-launch``` module instead (and remove the duped variable occurrences).

**Comment by iankko** [comment]
@rcernich All requested changes done AFAICT. Intentionally left the last commit un-squashed for more straightforward illustration, what was done. Will squash once re-reviewed.

**Comment by jmtd** [comment]
> @jmtd does this look good to you?

@rcernich yup!

**Pull request merged**